client/components/migration/feature-announcement.jsx

```diff
@@ -0,0 +1,133 @@
+/**
+ * External dependencies
+ */
+import React from 'react';
+import { Flex, FlexItem, Modal, Icon, Button } from '@wordpress/components';
+import { useCallback, useState } from '@wordpress/element';
+import { connect } from 'react-redux';
+import { bindActionCreators } from 'redux';
+import { localize } from 'i18n-calypso';
+
+/**
+ * Internal dependencies
+ */
+import bg from './images/wcshipping-migration.jpg';
+import { Dashboard, Preformatted, LessonPlan, Shipping } from './icons';
+import {
+	isEligableToMigrate,
+} from 'woocommerce/woocommerce-services/state/shipping-label/selectors';
+
+const FeatureAnnouncement = ({ translate, isEligable }) => {
+	const [isOpen, setIsOpen] = useState(isEligable);
+	const [isUpdating, setIsUpdating] = useState(false);
+
+	const closeModal = useCallback(() => {
+		setIsOpen(false);
+	});
+
+	const snooze = useCallback(() => {
+		// Todo: implement maybe later
+	});
+
+	const update = useCallback(() => {
+		// Todo: implement update
+		setIsUpdating(true);
+		setTimeout(() => setIsUpdating(false), 2000);
+	});
+
+	return <>{isOpen && (<Modal
+		className="migration__announcement-modal"
+		shouldCloseOnEsc={false}
+		shouldCloseOnClickOutside={false}
+		onRequestClose={closeModal}
+	>
+		<Flex gap={8}>
+			<FlexItem>
+				<Flex>
+					<span>
+						{translate('Update')}
+					</span>
+					<h2>
+						{translate('A new dedicated WooCommerce Shipping extension is now available')}
+					</h2>
+					<p>{translate(
+						'WooCommerce Shipping and WooCommerce Tax are now two dedicated extensions. We\'ll automatically deactivate WooCommerce Shipping & Tax and carry over your settings when you update.')}< /p>
```

Comment:
🚧 – Request changes – “It’s quite a blocking problem.”

`JSX element 'p' has no corresponding closing tag.ts(17008)` From my editor

Reply:
the `p` tag is being closed and my IDE doesn't nag about it and it builds and works fine.
![yQbvLG.png](https://github.com/Automattic/woocommerce-services/assets/789421/20d27eed-4746-4acb-a2e5-e8a76028cfc0)

Reply:
Ok, then let's stick with your editor.

Reply:
I had a 1:1 chat with @samnajian. Adding some clarity to the issue here (discovered by my VSCode editor.

The issue here is `< /p>` gives me the following errors:
![image](https://github.com/Automattic/woocommerce-services/assets/572862/c3c366f4-42b9-4fdb-8116-d0e8c951b87b)
![image](https://github.com/Automattic/woocommerce-services/assets/572862/00fbed35-c457-436b-907f-3f3ade827236)

The fix is to update `< /p>` to `</p>` but I can fix that later when I work on this file.